utils/lib_static.sh

```diff
@@ -4,16 +4,16 @@
 
 STATIC_BUILD_COMMIT="[build] /static"
 STATIC_BUILT_PATHS=(
-    searx/static/themes/oscar/css
-    searx/static/themes/oscar/js
-    searx/static/themes/oscar/src/generated/pygments-logicodev.less
-    searx/static/themes/oscar/src/generated/pygments-pointhi.less
-    searx/static/themes/simple/css
-    searx/static/themes/simple/js
-    searx/static/themes/simple/src/generated/pygments.less
-    searx/static/themes/simple/img
-    searx/templates/__common__/searxng-wordmark.min.svg
-    searx/templates/simple/icons.html
+    'searx/static/themes/oscar/css/*.css'
+    'searx/static/themes/oscar/js/*.js'
+    'searx/static/themes/oscar/src/generated/pygments-logicodev.less'
+    'searx/static/themes/oscar/src/generated/pygments-pointhi.less'
+    'searx/static/themes/simple/css/*.css'
+    'searx/static/themes/simple/js/*.js'
+    'searx/static/themes/simple/src/generated/pygments.less'
+    'searx/static/themes/simple/img'
+    'searx/templates/__common__/searxng-wordmark.min.svg'
+    'searx/templates/simple/icons.html'
```

Comment:
In commit 4704efa3 and commit ab454742 I removed the .map files from the repository.

The files are created by `themes.all` but marked as ignored in the .gitignore files in the css & js folders.

With we have the .map for locally debug purpose, but do not deploy them under the /static URL.  BTW: this minimize the commits from `make static.build.commit`.

Reply:
How the browser will find the .map files?

Reply:
If you run (e.g.) `make themes.all`  the .map files are where they are .. nothing changed. But there is no need to commit the .map files .. hope I was clear / otherwise ask.

Reply:
I wish to keep the .map files in the build because the day there is an error on an instance (not on localhost), it is easy to debug.
Also, the AGPL3 asks to provide the source code.

Reply:
> Also, the AGPL3 asks to provide the source code.

The source code is public here at github / no need to serve sources on every instance / BTW python sources are not served online in the instance.

> I wish to keep the .map files in the build because the day there is an error on an instance (not on localhost), it is easy to debug.

Really, did you ever had the need for? / I'm not familiar with, is it common to serve debug artefacts on a production server? 

If you want the map files back, we can leave it as is, but what me rumbles is the git-commit noise on the static folder caused by .map files. 

E.g have a look at the history of https://github.com/searxng/searxng/commits/master/searx/static/themes/oscar/css/logicodev.min.css.map

This .map file is changed every time `make static.build.commit` is called, even if nothing has changed in the _logicodev style_ since month // the symbols in this file are written in an unsorted manner, what changes the file, each time the file is newly generated by the grunt task.

Reply:
> The source code is public here at github / no need to serve sources on every instance

Few years ago, some public instances added some tracking. That's why there is an HTML grade on searx.space. The .map files allows to evaluate if a change is harmful or not for the privacy. It doesn't provide a clear answer, but it's help. In addition if the .map files are removed, may be there is a catch.

Look at searx.space, search for `Cjs` on the HTML grade filter, some instances don't provide the source code (= the hashes of the downloaded files don't match the hahes of the git repository).

It is matter of transparency IMO.

> Really, did you ever had the need for? / I'm not familiar with, is it common to serve debug artefacts on a production server?

To understand the CSP issues. Example on your instance to make sure what is going on about the autocompletion & the CSP headers.

---

The topic of this PR is to remove 4 unused files (2 js, 2 css), nothing more.

Reply:
> It is matter of transparency IMO.

OK I understand

> The topic of this PR is to remove 4 unused files (2 js, 2 css), nothing more.

OK, will remove from the branch, what is not a part of this topic ..